src/libs/CollectionUtils.ts

```diff
@@ -0,0 +1,10 @@
+function lastItem<T>(object: Record<string, T> = {}): T {
```

Comment:
```suggestion
/**
 * Return the highest item in a numbered collection
 *
 * e.g. {1: '1', 2: '2', 3: '3'} -> '3'
 */
function lastItem<T>(object: Record<string, T> = {}): T | undefined {
```

Leave JSDoc comments.

And we should return `T | undefined` because there is possibility of returning nothing.